cpp/src/sampling/neighbor_sampling_impl.hpp

```diff
@@ -252,11 +337,6 @@ neighbor_sample_impl(raft::handle_t const& handle,
 
     starting_vertices =
       raft::device_span<vertex_t const>(frontier_vertices.data(), frontier_vertices.size());
```

Comment:
I assume we don't need this code anymore. We are directly creating a span from `frontier_vertices`.

Reply:
Should I initialize `frontier_vertices` to `starting_vertices`?